client/src/api/investments.js

```diff
@@ -1,11 +1,5 @@
 import httpClient from './httpClient';
 
-//EG request
 export const addInvestment = (value, projectId) => {
-    return httpClient.post(`/investments/${projectId}`, {
-        {
-            "value": value,
-            "projectId": projectId
-        }
-    });
+    return httpClient.post(`/investments/${projectId}`, { value, projectId });
```

Comment:
I don't think this was never used -- was just an example of the route to backend (EG request)